legistar/bills.py

```diff
@@ -214,9 +214,7 @@ def dateBound(creation_date):
 
 
 class LegistarAPIBillScraper(LegistarAPIScraper):
-    # Make parameter optional, as it is in events.py
-    def matters(self, since_datetime=None):
-
+    def matters(self, since_datetime=None, scrape_restricted=False):
```

Comment:
I think we should just set a `scrape_restricted` property on the object, and not change these signatures.